subsys/bluetooth/audio/bap_broadcast_sink.c

```diff
@@ -1128,4 +1117,15 @@ static int broadcast_sink_init(void)
 	return 0;
 }
 
+struct bt_bap_broadcast_sink *broadcast_sink_get_by_pa(struct bt_le_per_adv_sync *sync)
```

Comment:
What was the reasoning behind moving this function?

Reply:
@Thalley I was trying to keep non-static functions and the bottom of the file

Reply:
Ah, that makes sense. That's fine :)